emperor/support_files/js/controller.js

```diff
@@ -66,23 +94,39 @@ EmperorController.prototype.resize = function(width, height){
   // the area we have to present the plot is smaller than the total
   var plotWidth = this.$plotSpaceId.width();
 
+  // TODO: The below will need refactoring
+  // This is addressed in issue #414
```

Comment:
If I remove this line, can I reopen the issue?  I think that refactoring is critical and I want to have a reminder somewhere